ui/src/app/services/error-handling.service.ts

```diff
@@ -73,7 +73,9 @@ export class ErrorHandlingService {
             this.setServerError();
             throw e;
           case 403:
-            this.setUserDisabledError();
+            if (JSON.parse(e._body).code === 'User Disabled') {
```

Comment:
Is `e._body` guaranteed to be present and non-empty? Possibly should guard against that being missing (unless this is guarantee by Angular or our swagger codegen).

Reply:
e._body will always be present _as long as we include any body in our errors._

I think any error we through should at least have a message included, so I feel pretty safe doing this, but if you would prefer to have a null check first, thats totally also doable.

Reply:
Thanks for checking on that. I'd prefer a null/undef check first in that case. This hooks into every outgoing XHR via the Angular stack, so in theory this error may not even have come from the Workbench API. There are also places in the Workbench API where we do things like forward errors from FC, where I'm less confident we'll always set an error body (though I think we probably are).